lib/widgets/message_list.dart

```diff
@@ -277,7 +277,7 @@ class _MessageListPageState extends State<MessageListPage> implements MessageLis
             narrow: ChannelNarrow(streamId)))));
     }
 
-    return Scaffold(
+    return PageRoot(child: Scaffold(
```

Comment:
This probably deserves a brief comment saying why this needs a PageRoot here. (It's so that the PageRoot context can be used to find the MessageListPageState.)